holidays/ch.yaml

```diff
@@ -0,0 +1,129 @@
+  _state_code: bl
+
+Basel-Stadt:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Basel-Stadt&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: bs
+
+Bern:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Bern&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: be
+
+Freiburg:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Freiburg&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: fr
+
+Genf:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Genf&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: ge
+
+Glarus:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Glarus&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: gl
+
+Graubünden:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Graub%C3%BCnden&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: gr
+
+Jura:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Jura&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: ju
+
+Luzern:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Luzern&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: lu
+
+Neuenburg:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Neuenburg&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: ne
+
+Nidwalden:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Nidwalden&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: nw
+
+Obwalden:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Obwalden&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: ow
+
+Schaffhausen:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Schaffhausen&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: sh
+
+Schwyz:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Schwyz&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: sz
+
+Solothurn:
+  _nominatim_url: 'https://nominatim.openstreetmap.org/search?format=json&country=Switzerland&state=Solothurn&zoom=18&addressdetails=1&limit=1&accept-language=de,fr,it,rm'
+  _state_code: so
+
+St. Gallen:
```

Comment:
The name here should match the one used above ("Sankt Gallen"). It the current state. this is not a bug in that it breaks anything a user would notice but it will cause an issue as soon as somebody tries to define some overwrites here.

Please check this for the other cantons as well.

Reply:
That should be fixed in the corresponding fixup-commit 0a02862d3. I did not notice any other, similar inconsistencies.